extensions/amp-animation/0.1/web-animations.js

```diff
@@ -355,35 +461,22 @@ export class MeasureScanner extends Scanner {
       fill: WebAnimationTimingFill.NONE,
     };
 
-    /** @private {!Array<!Element> } */
-    this.targets_ = [];
-  }
+    /** @private {!Array<!InternalWebAnimationRequestDef>} */
+    this.requests_ = [];
 
-  /**
-   * @param {!../../../src/service/resources-impl.Resources} resources
-   * @return {!Promise<!WebAnimationRunner>}
-   */
-  createRunner(resources) {
-    if (getMode().localDev || getMode().development) {
-      user().fine(TAG, 'Animation: ', this.requests_);
-    }
-    return this.unblockElements_(resources).then(() => {
-      return new WebAnimationRunner(this.requests_);
-    });
+    /** @const @private {!Array<!Promise>} */
+    this.deps_ = [];
```

Comment:
Dependencies to fully resolve all animation requests. Some can only be done asynchronously. I added the description.